### PR TITLE
Create a make command to create a report with x amount of host.

### PR DESCRIPTION
**Pull request by myersCody**

### CODE REVIEW

File: scripts/create_report.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+            for x in range(0, num_gen):
+                ip = socket.inet_ntoa(struct.pack('>I', random.randint(1, 0xffffffff)))
+                ip_addrs.append(ip)
+            host['ip_addresses'] = ip_addrs
+        uuids = ['bios_uuid', 'etc_machine_id', 'insights_client_id', 'subscription_manager_id']
+        for uid in uuids:
+            if host.get(uid, None):
+                host[uid] = str(uuid.uuid4())
+        fqdn = host.get('fqdn', None)
+        if fqdn:
```
Comment (abaiken) [inline scripts/create_report.py]: we could do this if clause the same way you did ip addresses and mac addresses:
 `if host.get('fqdn', None):`

File: scripts/create_report.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        open(meta_path, 'w').write(json.dumps(self.metadata, indent=4))
+
+
+if __name__ == "__main__":
+    try:
+        args = sys.argv
+        hosts_arg = list(filter(lambda x: 'hosts' in x, sys.argv))[0]
+        host_num = int(hosts_arg.split('=')[1])
+    except Exception as e:
+        print('Inproper value passed in for hosts.')
```
Comment (abaiken) [inline scripts/create_report.py]: Change `Inproper` to `Improper`

File: scripts/create_report.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+            self.fqdn_count += 1
+            new_fqdn = 'host_number_%s' % self.fqdn_count
+            host['fqdn'] = new_fqdn
+        return host
+
+    def create_slices(self):
+        number_hosts = self.target_hosts
+        if number_hosts % self.slice_max:
+            number_of_slices = number_hosts // self.slice_max + 1
+            hosts_per_slice = number_hosts // number_of_slices + 1
```
Comment (abaiken) [inline scripts/create_report.py]: I think that this is adding on an extra host for any number of hosts that isn't perfectly divisible by 10000. For example lets take 20 hosts: 
```
number_of_ slices = 20 // 10000 + 1 = 1 
hosts_per_slice = 20 // 1 +1 = 21 
```

Comment (myersCody) [reply scripts/create_report.py]: Good catch! I wonder if we have the same bug for quipucords: https://github.com/quipucords/quipucords/blob/7858056bbbdffe11490b8e5a1fc1fb8c37cb6a06/quipucords/api/insights_report/view.py#L99

Granted the previous implementation was splitting off portion of a giant list of host so the additional 1 might of been needed for that. It is just not needed for what I was doing and didn't catch it.

**Pull request merged**